src/encoded/static/components/filegallery.js

```diff
@@ -3678,6 +3678,13 @@ class FileGalleryRendererComponent extends React.Component {
             meta = globals.graphDetail.lookup(this.state.infoNode)(this.state.infoNode, this.handleNodeClick, auditIndicators, auditDetail, this.context.session, this.context.sessionProperties);
         }
 
+        let defaultLocation = null;
+        if (context.examined_regions) {
+            defaultLocation = context.examined_regions[0];
+        } else if (context.examined_loci && context.examined_loci[0].gene && context.examined_loci[0].gene.locations) {
```

Comment:
You could use the ?. syntax to simplify this, though I’d guess transpiling converts it to what you have.